docs/java-rest/high-level/ml/set-upgrade-mode.asciidoc

```diff
@@ -0,0 +1,38 @@
+--
+:api: set-upgrade-mode
+:request: SetUpgradeModeRequest
+:response: AcknowledgedResponse
+--
+[id="{upid}-{api}"]
+=== Set Upgrade Mode API
+
+The Set Upgrade Mode API provides the ability pause all {ml} jobs and {dfeeds}.
```

Comment:
I’m not sure “pause” is the right word here. It makes the endpoint sound innocuous when in reality it will cause considerable churn in the cluster. Or maybe we can use “pause” but add an extra sentence to elaborate that this is not an endpoint to call without good reason. I’ll think more carefully about this tomorrow.

Reply:
Maybe something like this?

> The Set Upgrade Mode API provides the ability to temporarily halt all {ml} jobs and {dfeeds} while leaving their reported state unaltered.  This means that exiting upgrade mode can return all jobs and {dfeeds} to the desired state once the portions of the upgrade process that would cause them to fail is complete.

That's a bit of a mouthful so maybe @lcawl can think of a way to say the same thing more elegantly.